Assets/_Scripts/3Managers/Logger.cs

```diff
@@ -0,0 +1,23 @@
+using Assets._Scripts._3Managers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Logger : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject LogPrefab;
+
+    public enum LogType
+    {
+        Error,
+        Debug,
```

Comment:
What's debug for? I would expect: Error, Warning, Info/Message

Reply:
For us